### PR TITLE
Feature/13029 ess u i show exchange set progress screen and guestimated size

### PR DESCRIPTION
**Pull request by Rajesh14881**
Added Estimated Size with Spinner

### CODE REVIEW

**Review by rvijayavel — APPROVED** [review]
Looks good.

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Please resolve merge conflicts

File: Tests/UnitTests/ess-download-exchangeset.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    })
+    .compileComponents();
+  });
+
+  beforeEach(() => {
+    AppConfigService.settings = {
+      essConfig: {
+      MaxEncLimit: 100,
+      MaxEncSelectionLimit : 5,
+      avgSizeofENC: 0.3
```
Comment (crdandoukho) [inline Tests/UnitTests/ess-download-exchangeset.component.spec.ts:52]: Please change to `avgSizeofENCinMB`
```suggestion
      avgSizeofENCinMB: 0.3
```

**Review by crdandoukho — CHANGES_REQUESTED** [review]
Only a very small change needed please

**Pull request merged**